docs/extend/plugins_logging.md

```diff
@@ -46,76 +36,77 @@ receiving logs for.
 Logs will be streamed over the defined file in the request. On Linux this file
 is a FIFO. Logging plugins are not currently supported on Windows.
 
-**Request**:
-```json
-{
-		"File": "/path/to/file/stream",
-		"Info": {
-			"ContainerID": "123456"
-		}
-}
-```
-
-`File` is the path to the log stream that needs to be consumed. Each call to
-`StartLogging` should provide a different file path, even if it's a container
-that the plugin has already received logs for prior. The file is created by
-docker with a randomly generated name.
-
-`Info` is details about the container that's being logged. This is fairly
-free-form, but is defined by the following struct definition:
-
-```go
-type Info struct {
-	Config              map[string]string
-	ContainerID         string
-	ContainerName       string
-	ContainerEntrypoint string
-	ContainerArgs       []string
-	ContainerImageID    string
-	ContainerImageName  string
-	ContainerCreated    time.Time
-	ContainerEnv        []string
-	ContainerLabels     map[string]string
-	LogPath             string
-	DaemonName          string
-}
-```
-
-
-`ContainerID` will always be supplied with this struct, but other fields may be
-empty or missing.
-
-**Response**
-```json
-{
-	"Err": ""
-}
-```
-
-If an error occurred during this request, add an error message to the `Err` field
-in the response. If no error then you can either send an empty response (`{}`)
-or an empty value for the `Err` field.
-
-The driver should at this point be consuming log messages from the passed in file.
-If messages are unconsumed, it may cause the container to block while trying to
-write to its stdio streams.
-
-Log stream messages are encoded as protocol buffers. The protobuf definitions are
-in the
-[docker repository](https://github.com/docker/docker/blob/master/api/types/plugins/logdriver/entry.proto).
-
-Since protocol buffers are not self-delimited you must decode them from the stream
-using the following stream format:
-
-```
-[size][message]
-```
-
-Where `size` is a 4-byte big endian binary encoded uint32. `size` in this case
-defines the size of the next message. `message` is the actual log entry.
-
-A reference golang implementation of a stream encoder/decoder can be found
-[here](https://github.com/docker/docker/blob/master/api/types/plugins/logdriver/io.go)
+- Request
```

Comment:
Slight bit on the fence on using bullet-lists for this; do you have an example what it looks like when rendered in the docs? Does it significantly improve readability?

Reply:
not super happy with the bullet list either but we shouldn't use a bolded paragraph as a heading. Here's what it currently looks like: 
<img width="822" alt="image" src="https://github.com/docker/cli/assets/35727626/7804c4ae-1294-4b65-993d-a66e0f6d1b60">

Reply:
I don't want to make them actual headings either... so without actually restructuring this stuff more this was the best I could come up with. I think it's fine, especially for these pages.

Reply:
Would "plain" (non-bold) work? e.g.


```
Request example:

```json
{
  "File": "/path/to/file/stream"
}
```

On a successful request, the `Err` field is left empty:

```json
{
  "Err": ""
}
```

If an error occurred .......

```json
{
  "Err": "an error occurred"
}
```

Reply:
yeah that would work. I can change it